### DISCUSSION TITLE
How can I customize existing themes with Azure segment showing subscription after entering "az", like in the default theme?

### DISCUSSION THREAD

**Discussion by nlykkei** [JanDeDobbeleer/oh-my-posh#6056] [General]
I want to use a custom theme, like amro, but it doesn't support displaying the Azure segment when entering "az" in the terminal.

The default theme supports this (see below). I have looked for this theme to use it for inspiration, but I can't find it.

Could someone point me in the direction of the default theme, so I can copy-paste the Azure segment from there, or show how to modify the amro theme to display Azure segment on entering "az"?

https://ohmyposh.dev/docs/themes#amro
https://ohmyposh.dev/docs/segments/cloud/az

<img width="444" alt="image" src="https://github.com/user-attachments/assets/377b86c6-4902-48d6-86fd-7c80b49276c7" />

Bonus:

Is this a theme you've seen before?

<img width="837" alt="image" src="https://github.com/user-attachments/assets/cdaa9737-02b1-4740-a4d1-7eae15379491" />

**Comment by JanDeDobbeleer** [comment]
@nlykkei you're looking for [tooltips](https://ohmyposh.dev/docs/configuration/tooltips). [Customize](https://ohmyposh.dev/docs/installation/customize#custom-configuration) amro to include those tooltips.
